src/js/view/controls/controlbar.js

```diff
@@ -485,7 +485,9 @@ export default class Controlbar {
         const dvrMode = streamType === 'DVR';
 
         // Hide rewind button when in LIVE mode
-        this.elements.rewind.toggle(!liveMode);
+        if (this.elements.rewind) {
+            this.elements.rewind.toggle(!liveMode);
+        }
```

Comment:
It would be cool to use @babel/plugin-proposal-optional-chaining so instead of this if statement, we could just do `this.elements.rewind?.toggle(!liveMode);`